src/main/java/persistence/sql/ddl/FieldMetadataExtractor.java

```diff
@@ -0,0 +1,67 @@
+package persistence.sql.ddl;
+
+import jakarta.persistence.Column;
+
+import java.lang.reflect.Field;
+
+public class FieldMetadataExtractor {
+
+    private final Field field;
+
+    public FieldMetadataExtractor(Field field) {
+        this.field = field;
+    }
+
+    public String getDefinition() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(getFieldName(field));
+        sb.append(" ");
+        sb.append(map(field.getType()));
+        sb.append(" ");
+        sb.append(getColumnOptionValue());
+
+        if (sb.lastIndexOf(" ") == sb.length() - 1) {
+            sb.deleteCharAt(sb.length() - 1);
+        }
```

Comment:
```suggestion
        if (sb.lastIndexOf(" ") == sb.length() - 1) {
            sb.deleteCharAt(sb.length() - 1);
        }
```
우리만의 `어떠한Builder` 가 있다면 해당 책임을 가질 수 있지 않을까요?